flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
```

Comment:
DESIGN_NICKNAME already defaults to DESIGN_NAME so there is no need to set it here.

Reply:
@maliberty copied from ASAP7 built-in examples and just modified as per my design requirement.
Only changed DESIGN NAME and DIE AREA

Reply:
That is because the ASAP7 platform is kind of a mess right now. We should catch these problems where we can.

Reply:
Why are `DESIGN_NAME` and `DESIGN_NICKNAME` different here? It is honestly more worth renaming the source folder to `ethmac` and removing `DESIGN_NICKNAME`

Reply:
@rovinski as per @maliberty  suggestion if module and directory name same. Will define DESIGN_NAME only, DESIGN_NICKNAME already defined in Makefile. I will update this all design config.mk